app/main.py

```diff
@@ -1,3 +1,18 @@
-def shop_trip():
-    # write your code here
-    pass
+import app.auxilaries.parsing_shopping as parsing
+
+
+def shop_trip() -> None:
+    data = parsing.parse_data_from_json()
+    customers = parsing.create_customer_classes(data[1])
+    shops = parsing.create_shops_classes(data[2])
```

Comment:
You could consider unpacking on a previous step to not refer to the data by index.